cgmes/cgmes-import-postprocessor/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+          <groupId>com.powsybl</groupId>
+          <artifactId>powsybl-cgmes</artifactId>
+          <version>6.9.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>powsybl-cgmes-import-postprocessor</artifactId>
+    <name>CGMES import postprocessors</name>
+    <description>Postprocessors for CGMES import</description>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <Automatic-Module-Name>com.powsybl.cgmes.importer.postprocessor</Automatic-Module-Name>
+                        </manifestEntries>
+                    </archive>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
```

Comment:
What is the purpose of this maven module which depends of so many other module?
We still have several modules for each of the post processor but why having this aggregator on top of others ?